app/src/main/java/ch/heigvd/pro/b04/android/Home/EmojiGridAdapter.java

```diff
@@ -3,22 +3,28 @@
 import android.view.LayoutInflater;
 import android.view.ViewGroup;
 import android.widget.ImageView;
+import android.widget.TextView;
 
 import androidx.annotation.NonNull;
 import androidx.cardview.widget.CardView;
+import androidx.lifecycle.LifecycleOwner;
 import androidx.recyclerview.widget.RecyclerView;
 
+import ch.heigvd.pro.b04.android.Poll.PollViewModel;
 import ch.heigvd.pro.b04.android.R;
 
 public class EmojiGridAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
 
     private static final int VIEW_TYPE_HEADER = 0;
     private static final int VIEW_TYPE_EMOJI = 1;
+    private static final int VIEW_TYPE_INSTRUCTION = 2;
 
     private HomeViewModel state;
+    private LifecycleOwner lifecycleOwner;
 
-    public EmojiGridAdapter(HomeViewModel state) {
+    public EmojiGridAdapter(HomeViewModel state, LifecycleOwner lifecycleOwner) {
         this.state = state;
+        this.lifecycleOwner = lifecycleOwner;
     }
```

Comment:
(if we don't need the `LiveData` observation, we could remove the `LifecycleOwner` from this `Adapter`.